jest.config.js

```diff
@@ -0,0 +1,27 @@
+const { generateWebpackAliasesFromTSConfig } = require('./webpackConfigUtils');
+
+const aliases = generateWebpackAliasesFromTSConfig()
+
+const moduleNameMapper = Object.keys(aliases).reduce((acc, alias) => ({
+  ...acc,
+  [`^${alias}/(.*)$`]: `${aliases[alias]}/$1`,
+}), {
+  '\\.(svg|ttf)$': '<rootDir>/__mocks__/fileMock.js',
+  '\\.(css)$': '<rootDir>/__mocks__/styleMock.js',
```

Comment:
Using a mock here would prevent to know if our imports are not correctly pointing to a real file?